cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -28,14 +28,14 @@
 
 VISIUM_AND_IS_SINGLE_TRUE_MATRIX_SIZE = 4992
 SPATIAL_HIRES_IMAGE_MAX_DIMENSION_SIZE = 2000
-SPATIAL_HIRES_IMAGE_MAX_DIEMSNION_SIZE_VISIUM_11MM = 4000
+SPATIAL_HIRES_IMAGE_MAX_DIMENSION_SIZE_VISIUM_11MM = 4000
 
 CONDITION_IS_VISIUM = "a descendant of 'EFO:0010961' (Visium Spatial Gene Expression)"
 CONDITION_IS_SEQV2 = f"'{ASSAY_SLIDE_SEQV2}' (Slide-seqV2)"
 
 
-ERROR_SUFFIX_VISIUM = f"obs['assay_ontology_term_id'] is either {CONDITION_IS_VISIUM} or {CONDITION_IS_SEQV2}"
-ERROR_SUFFIX_VISIUM_AND_IS_SINGLE_TRUE = f"{ERROR_SUFFIX_VISIUM} and uns['spatial']['is_single'] is True"
+ERROR_SUFFIX_SPATIAL = f"obs['assay_ontology_term_id'] is either {CONDITION_IS_VISIUM} or {CONDITION_IS_SEQV2}"
+ERROR_SUFFIX_VISIUM_AND_IS_SINGLE_TRUE = f"{ERROR_SUFFIX_SPATIAL} and uns['spatial']['is_single'] is True"
```

Comment:
sorry--should have caught this on the first pass, but all these ERROR_SUFFIX_VISIUM_* error messages are specifically for visium + its descendants, and are not meant to include CONDITION_IS_SEQV2 (example--see [requirements for spatial[library_id] in uns](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/drafts/5.3.0.md#spatiallibrary_id))

so, there should be a separate ERROR_SUFFIX_VISIUM in addition to ERROR_SUFFIX_SPATIAL, that does not include CONDITION_IS_SEQV2. And that ERROR_SUFFIX_VISIUM is what ERROR_SUFFIX_VISIUM_AND_IS_SINGLE_TRUE should f-string in.